GitUI/CommandsDialogs/SettingsDialog/Pages/GitConfigAdvancedSettingsPage.Designer.cs

```diff
@@ -31,47 +31,60 @@ private void InitializeComponent()
             this.checkBoxRebaseAutostash = new System.Windows.Forms.CheckBox();
             this.checkBoxFetchPrune = new System.Windows.Forms.CheckBox();
             this.checkBoxPullRebase = new System.Windows.Forms.CheckBox();
+            this.checkBoxRebaseAutosquash = new System.Windows.Forms.CheckBox();
             this.SuspendLayout();
             // 
             // checkBoxRebaseAutostash
             // 
             this.checkBoxRebaseAutostash.AutoSize = true;
-            this.checkBoxRebaseAutostash.Location = new System.Drawing.Point(19, 80);
+            this.checkBoxRebaseAutostash.Location = new System.Drawing.Point(19, 82);
             this.checkBoxRebaseAutostash.Name = "checkBoxRebaseAutostash";
-            this.checkBoxRebaseAutostash.Size = new System.Drawing.Size(228, 17);
+            this.checkBoxRebaseAutostash.Size = new System.Drawing.Size(363, 19);
             this.checkBoxRebaseAutostash.TabIndex = 3;
-            this.checkBoxRebaseAutostash.Text = "Automatically stash before doing a rebase";
+            this.checkBoxRebaseAutostash.Text = "Automatically stash before doing a rebase i.e. `rebase.autoStash`";
```

Comment:
??
table?
```suggestion
            this.checkBoxRebaseAutostash.Text = "Automatically stash before doing a rebase [rebase.autoStash]";
```

Reply:
I like the idea and that allows me to do a refactoring that doesn't change the existing translation strings.
Could you review again? I have pushed a new commit but it's maybe easier to review "squashed". I will squash them if review is OK.

Reply:
Looks good!